vault/quotas/quotas.go

```diff
@@ -1015,8 +1027,10 @@ func Load(ctx context.Context, storage logical.Storage, qType, name string) (Quo
 // Setup loads the quota configuration and all the quota rules into the
 // quota manager.
 func (m *Manager) Setup(ctx context.Context, storage logical.Storage, isPerfStandby, isDRSecondary bool) error {
```

Comment:
Why doesn't this one need dbAndCacheLock?

Reply:
Good point - I simply missed it. Adding it